fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationsDialogFragment.kt

```diff
@@ -58,6 +62,9 @@ class TranslationsDialogFragment : BottomSheetDialogFragment() {
     private val translationDialogBinding = ViewBoundFeatureWrapper<TranslationsDialogBinding>()
     private lateinit var translationsDialogStore: TranslationsDialogStore
     private var isTranslationInProgress: Boolean? = null
+    private val downloadLanguagesFeature =
```

Comment:
Could we organize these variables a bit better? Just moving `ViewBoundFeatureWrapper` together would be a good start.

Reply:
Done